.licensed.yaml

```diff
@@ -0,0 +1,19 @@
+sources:
+  go: true
+allowed:
+  - mit
+  - apache-2.0
+  - mpl-2.0
+  - bsd-2-clause
+  - bsd-3-clause
+  - isc
+reviewed:
+  go:
+    - github.com/magiconair/properties # bsd-2
+    - github.com/pmezard/go-difflib/difflib # bsd-style
+    - golang.org/x/**/* # bsd-style
+    - gopkg.in/ini.v1 # apache-2.0
+    - gopkg.in/yaml.v2 # apache-2.0
+ignored:
+  go:
+    - github.com/kanopy-platform/k8s-auth-portal/**/* # no-license
```

Comment:
nit: we should either add a license to this repo, or better yet not import from this repo, rather than just ignore.